packages/outputarea/style/base.css

```diff
@@ -73,6 +73,17 @@
   width: var(--jp-cell-prompt-width);
   height: 100%;
   opacity: 0.5;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.jp-OutputArea-promptOverlay .jp-icon-output {
+  display: none;
+}
+
+.jp-OutputArea-promptOverlay:hover .jp-icon-output {
+  display: initial;
```

Comment:
Could you drop the cursor in this PR? It is just a matter of removing the cursor style rules line 92 and line 96 below.